src/Core/SharedKernels/Questionnaire/WB.Core.SharedKernels.Questionnaire/Api/AttachmentContent.cs

```diff
@@ -39,4 +39,9 @@ public class QuestionnaireLiteInfo
         public DateTime LastUpdateDate { get; set; }
     }
 
+    public class QuestionnaireSettings
+    {
+        public bool CanSaveScenario { get; set; }

```

Comment:
What about to use flag like IsAnonymousMode and use policy depending on this flag?